datafusion/sqllogictest/test_files/case.slt

```diff
@@ -50,7 +50,7 @@ NULL
 6
 NULL
 NULL
-7
+NULL
```

Comment:
Can you also please add a test to this file where there are no nulls (to ensure the codepath where `null_count = 0` is also covered)

Reply:
done